### ISSUE TITLE
Fingerprint hardware missing 

### ISSUE THREAD

**Issue by Spako0147** [SmartPack/PackageManager#84]

**Comment by sunilpaulmathew** [comment]
@Spako0147 Can you configure fingerprint first, and then check the settings page of this app?

**Comment by Spako0147** [comment]
> @Spako0147 Can you configure fingerprint first, and then check the settings page of this app?

Yes but it will cost you $500 or you can send me a Google play gift worth the same amount on Whatsapp.

**Comment by sunilpaulmathew** [comment]
@Spako0147 
Okay, closing this issue as I am not a magician to enable fingerprint on a device without hardware support.

**Comment by Spako0147** [comment]
Why....?

**Comment by sunilpaulmathew** [comment]
> Why....?

I'm sorry. I got the confused with one of my another app which already have fingerprint support. I'm reopening this issue.. Still don't know it's important for a package manager app.
Sorry again & have a nice day
